tests/digest_tests.rs

```diff
@@ -47,7 +47,7 @@ fn digest_misc() {
 }
 
 // wasm_bindgen doesn't build this correctly.
-#[cfg(not(target_arch = "wsam32"))]
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
Thanks for pointing out the typo. I fixed this the right way in PR #1647, so you can remove this change.

Reply:
Okay, I will, Thanks for the review.